terraform/modules/environment_dns/dns.tf

```diff
@@ -224,11 +224,11 @@ resource "aws_route53_record" "brokered_mail_ns" {
 }
 
 locals {
-  external_domain               = "external-domains-${var.stack_name}.cloud.gov"
+  external_domain               = "external-domains-${var.stack_name}.cloud.gov."
   csb_helper_domain_name        = "services.${var.domain}."
-  csb_helper_domain_record      = "services.${var.domain}.${local.external_domain}"
+  csb_helper_domain_record      = "services.${var.domain}.${local.external_domain}."
```

Comment:
[issue] looks like this will result in a double ending `.` since you added it to `local.external_domain` also